client/src/templates/Challenges/redux/completion-epic.test.js

```diff
@@ -0,0 +1,31 @@
+import { TestScheduler } from 'rxjs/testing';
+import completionEpic from './completion-epic';
+import { submitChallenge, submitChallengeComplete } from './actions';
+
+describe('completionEpic', () => {
+  describe('signed out user', () => {
```

Comment:
What do you think about the signed-in user? Do they use the same pipeline? If yes, please add the test for them too.

Reply:
They do, yeah. It's a bit more complicated, though (I think a fair bit of mocking will be required).

Since deploying without this PR would introduce a bug, do you mind if I create a followup PR to expand the tests?

Reply:
Absolutely, it just caught my eye that fixing something may break something else.